lua/modules/hyperspace.i

```diff
@@ -580,9 +580,9 @@ playerVariableType playerVariables;
 %rename("%s") CommandGui::outOfFuel;
 %immutable CommandGui::outOfFuel;
 %rename("%s") CommandGui::bPaused;
-%immutable CommandGui::bPaused;
+// %immutable CommandGui::bPaused;
```

Comment:
This doesn't need to be commented, it can be fully removed.

Reply:
alrighty!